dev-env/bin/da-ghci

```diff
@@ -59,25 +59,38 @@ def main():
 
         # Discover the Haskell target.
         target = query_haskell_target(source_file)
+
+        # This isn't a haskell_repl target
+        is_repl = False
     else:
         # No specific module to load.
         module_name = None
 
         # Assume we were given a Bazel target.
         target = args.target
 
+        # Find out if it's a haskell_repl target
+        ret = subprocess.run(
```

Comment:
You should run `bazel fetch` before the first invocation of `bazel query`, otherwise this becomes flaky. See `query_haskell_target`.

Reply:
Thanks! I moved the `bazel query` to a separate function and run `bazel fetch` before `bazel query`.